src/java.base/share/classes/java/util/Optional.java

```diff
@@ -373,7 +373,7 @@ public Stream<T> stream() {
      * @throws NullPointerException if no value is present and the supplying
      *         function is {@code null}
      */
-    public T orElseGet(Supplier<? extends T> supplier) {
+    public @Nullable T orElseGet(Supplier<? extends @Nullable T> supplier) {
```

Comment:
Comment that this would ideally use `@PolyNull`?

Reply:
Done, thanks.